src/jira/client/jira-client.ts

```diff
@@ -421,6 +454,27 @@ const updateIssueKeysFor = (resources, func) => {
 	return resources;
 };
 
+/**
+ * Runs the mutatingFunc on the association values field for each entity resource
+ * Assumption is that the transformed resource only has one association which is for
+ * "issueIdOrKeys" association.
+ */
+const updateIssueKeyAssociationValuesFor = (resources: JiraRemoteLink[], mutatingFunc: any): JiraRemoteLink[] => {
+	resources?.forEach(resource => {
```

Comment:
is it possible that resources is undefined?  if that's the case, the type needs to be `resources?:JiraRemoteLink[]`

Reply:
Not really, we're checking for undefined before even calling this function (in [here](https://github.com/atlassian/github-for-jira/pull/712/files#diff-157b18999c0ae6ee36ed20dcfb16e01c5f9574ff54a27df8e5970db851b9bd11R13)) so this is just an extra check.